packages/chat/commands/crawl.ts

```diff
@@ -10,7 +10,7 @@ export default class Crawl extends AbstractCommand {
   }
 
   public name() {
-    return 'crawl'
+    return '_DEPRECATED_crawl'
```

Comment:
@kitschbent Please be aware we are adding a deprecated notice to the `crawl` command, just for the next release.

Reply:
feel free to ping me for details - we are removing crawl in favor of using the distributed hash table to disseminate routing information.